dataplane/server.go

```diff
@@ -130,14 +131,25 @@ func (d *Dataplane) InsertRoute(ctx context.Context, route *dpb.InsertRouteReque
 		return nil, status.Errorf(codes.InvalidArgument, "failed to parse prefix: %v", err)
 	}
 
-	isIPv4 := ipNet.IP.To4() != nil
+	ip := ipNet.IP.To4()
+	isIPv4 := true
+
 	var nextHopIP []byte
 	if nh := route.GetNextHops()[0].GetIp(); nh != "" {
```

Comment:
handle no nexthops (perhaps return an error since it's no different from a drop)

Reply:
done